tests/extmod/ssl_noleak.py

```diff
@@ -0,0 +1,40 @@
+# Ensure that SSL sockets can be allocated sequentially
+# without running out of available memory.
+try:
+    import io
+    import ssl
```

Comment:
Since this test doesn't run under CPython (and has a .exp) is it worth just doing `import tls`?

Also, could use unittest for this test (although it is nice and simple without unittest).

Reply:
Have refactored to use unittest, good idea.

We have to import the `ssl` module here in order to use `wrap_socket()`.